### ISSUE TITLE
The Collab notebook fails to run

### ISSUE THREAD

**Issue by soundling** [kingoflolz/mesh-transformer-jax#60]
Line 333 of transformer_shard.py is throwing this error since the last commit : 
"TypeError: fun_mapped() got an unexpected keyword argument 'return_logits'"

**Comment by stprior** [comment]
Looks like something to do with commit e0588fbba13488135656ebdf9cdac985c1d275fe.
As a workaround, you can check out the previous version by adding the second line below in the first cell

```
!git clone https://github.com/kingoflolz/mesh-transformer-jax.git
!cd mesh-transformer-jax/;git checkout 3ef2031b297ac1a82468a0fe88cd5e689aa6db1d
```

**Comment by kingoflolz** [comment]
fixed in #62
